develop.py

```diff
@@ -189,40 +208,62 @@ def shell():
     call_command('shell')
 
 
+class capture_all_output(object):
```

Comment:
Yup, removed.


On Sun, Jan 11, 2015 at 11:20 PM, Iacopo Spalletti <notifications@github.com
> wrote:

> In develop.py
> <https://github.com/divio/django-cms/pull/3611#discussion-diff-22770187>:
>
> > @@ -189,40 +208,62 @@ def shell():
> >      call_command('shell')
> >
> >
> > +class capture_all_output(object):
>
> This is now useless, right?
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/divio/django-cms/pull/3611/files#r22770187>.
>



-- 
*MARTIN KOISTINEN*


*BG GSM +359 878 787 253UK GSM +44 7703 046 154*
*UK VoIP +44 20 3286 9932*
*US VoIP +1 (**678) **389-6676*

*100% CARBON-NEUTRAL SINCE 1-JAN-2008*